src/replay_stage.rs

```diff
@@ -149,10 +152,12 @@ impl ReplayStage {
 
```

Comment:
seems even leader will try to send the vote.

Reply:
Yeah the goal was to have the leader's tvu participate in the network independently. Would this cause issues? I removed the ledger writing from tpu, assuming that only tvu will handle that now since it's always running.

Reply:
Confirmation time calculation "may" be incorrect, if leader is also voting. Not sure if there are any other issues.

@carllin removed the leader votes. He might have more insight into it.